### PR TITLE
Bah ct111 19025

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description 
As a developer, I need to update the WEAMS import into GIDS to process the STEM indicator so that the UI can display the STEM information on the school profile page.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19025

## Testing done

1. Run `bundle exec rake db:migrate`
2. Ensure the stem indicator column is present in the database
3. Run `bundle exec rails s -p 4000`
4. Open the GI Bill Comparison Tool Dashboard
5. Export Weam.csv and ensure stem indicator column is generated. 
6. Logged into the GIDS dashboard and exported a copy of the Weam CSV to ensure the stem indicator column was present. 

## Screenshots
Screen Shot 2019-06-24 at 9 26 03 AM

## Acceptance criteria
- [X] The weam.csv with "STEM Indicator" column is imported into GIDS

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: db/migrate/20190622019025_add_column_stem_indicator_to_weams.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddColumnStemIndicatorToWeams < ActiveRecord::Migration
+    def change
+      add_column :weams, :stem_indicator, :boolean
```
Comment (zurbergram) [inline db/migrate/20190622019025_add_column_stem_indicator_to_weams.rb]: should this default to false?

Comment (ksantiagomorgan6) [reply db/migrate/20190622019025_add_column_stem_indicator_to_weams.rb]: It doesn't specify in the story

**Review by sonntag-adam — APPROVED** [review]

File: db/migrate/20190622019025_add_column_stem_indicator_to_weams.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddColumnStemIndicatorToWeams < ActiveRecord::Migration
+    def change
+      add_column :weams, :stem_indicator, :boolean, default: false, null: false
```
Comment (zurbergram) [inline db/migrate/20190622019025_add_column_stem_indicator_to_weams.rb]: remove `default: false, null: false`

**Review by zurbergram — APPROVED** [review]
LGTM

**Comment by ksantiagoBAH** [comment]
@department-of-veterans-affairs/backend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by kfrz — APPROVED** [review]
:+1:

File: db/schema.rb [hunk 3]
```diff
@@ -305,6 +305,7 @@
     t.integer  "dod_bah"
     t.boolean  "approved",                                            default: false
     t.boolean  "vet_tec_provider",                                    default: false
+    t.boolean  "closure109"
```
Comment (jholton) [inline db/schema.rb]: Where did this come from? It's not reflected in the migration

File: db/schema.rb [hunk 4]
```diff
@@ -1114,6 +1115,12 @@
   add_index "scorecards", ["cross"], name: "index_scorecards_on_cross", using: :btree
   add_index "scorecards", ["ope"], name: "index_scorecards_on_ope", using: :btree
 
+  create_table "sec109_closed_schools", force: :cascade do |t|
```
Comment (jholton) [inline db/schema.rb]: Where did this come from? It's not reflected in the migration

**Review by jholton — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 3 above)
Comment (zurbergram) [reply db/schema.rb]: branch not updated with master

File: db/schema.rb (hunk 4 above)
Comment (zurbergram) [reply db/schema.rb]: branch not updated with master

**Review by jholton — APPROVED** [review]
LGTM

**Pull request merged**